docs/uninstall.md

```diff
@@ -0,0 +1,449 @@
+
+```
+kubectl delete -f https://storage.googleapis.com/knative-nightly/serving/latest/serving-default-domain.yaml
+```
+
+### Uninstalling a networking layer
+
+Follow the relevant procedure to uninstall the networking layer you installed:
+
+<!-- TODO: Link to document/diagram describing what is a networking layer.  -->
+<!-- This indentation is important for things to render properly. -->
+
+   {{< tabs name="serving_networking" default="Kourier" >}}
+   {{% tab name="Ambassador" %}}
+
+The following commands install Ambassador and enable its Knative integration.
+
+1. Uninstall Ambassador by running:
+
+   ```bash
+   kubectl delete --namespace ambassador \
+    -f https://getambassador.io/yaml/ambassador/ambassador-crds.yaml \
+    -f https://getambassador.io/yaml/ambassador/ambassador-rbac.yaml \
+    -f https://getambassador.io/yaml/ambassador/ambassador-service.yaml
+   ```
+
+1. Delete the Ambassador namespace:
+
+   ```bash
+   kubectl delete namespace ambassador
+   ```
+
+{{< /tab >}}
+
+{{% tab name="Contour" %}}
+
+The following commands install Contour and enable its Knative integration.
+
+1. Uninstall the Knative Contour controller by running:
+
+    ```bash
+    kubectl delete -f {{< artifact repo="net-contour" file="net-contour.yaml" >}}
+    ```
+
+1. Install a properly configured Contour:
+
+    ```bash
+    kubectl delete -f {{< artifact repo="net-contour" file="contour.yaml" >}}
+    ```
+
+{{< /tab >}}
+
+{{% tab name="Gloo" %}}
+
+Uninstall Gloo and the Knative integration by running:
+
+   ```bash
+   glooctl install knative --install-knative=false
+   ```
+
+{{< /tab >}}
+
+{{% tab name="Istio" %}}
+
+The following commands install Istio and enable its Knative integration.
+
+1. Uninstall the Knative Istio controller by running:
+
+    ```bash
+    kubectl delete -f {{< artifact repo="net-istio" file="net-istio.yaml" >}}
+    ```
+
+1. Optional: if you no longer need Istio, uninstall it by running:
+
+    ```bash
+    kubectl delete -f {{< artifact repo="net-istio" file="istio.yaml" >}}
+    ```
+
+{{< /tab >}}
+
+{{% tab name="Kong" %}}
+
+Uninstall Kong Ingress Controller by running:
+
+   ```bash
+   kubectl delete -f https://raw.githubusercontent.com/Kong/kubernetes-ingress-controller/0.9.x/deploy/single/all-in-one-dbless.yaml
+   ```
+
+{{< /tab >}}
+
+{{% tab name="Kourier" %}}
+
+Uninstall the Knative Kourier controller by running:
+
+   ```bash
+   kubectl delete -f {{< artifact repo="net-kourier" file="kourier.yaml" >}}
+   ```
+
+{{< /tab >}} {{< /tabs >}}
+
+
+### Uninstalling the Serving component
+
+1. Uninstall the Serving core components by running:
+
+    ```
+    kubectl delete -f https://storage.googleapis.com/knative-nightly/serving/latest/serving-core.yaml
+    ```
+
+1. Uninstall the required custom resources by running:
+
+    ```
+    kubectl delete -f https://storage.googleapis.com/knative-nightly/serving/latest/serving-crds.yaml
+    ```
```

Comment:
I believe the `core` files include all the `crd` files, so you shouldn't need to run both.

With that said, we might need to tear things down in a specific order; I'm not 100% sure here. @dprotaso might know, I'm not sure we've thought about it much.

Reply:
Okay, I'll leave things as they are for now.